### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by Menum** [johngodley/redirection#672]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.0.27
Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/redirect POST
Params: {"id":0,"url":"https://azurdancestudio.ro/608-2/","title":"","regex":false,"match_type":"url","action_type":"url","position":0,"group_id":1,"action_code":301,"action_data":{"url":"https://azurdancestudio.ro"}}
Code: 404 
Raw: <html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->

```

**Comment by johngodley** [comment]
Can you try the 3.1 beta please? https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by Menum** [comment]
Sure, let me try it and I will let you know.

**Comment by Menum** [comment]
Plugin: 3.1
WordPress: 4.9.2
PHP: 7.0.27
Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://azurdancestudio.ro/wp-json/

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/redirect POST
Params: {"id":0,"url":"https://azurdancestudio.ro/2r54325647457u6","title":"","regex":false,"match_type":"url","action_type":"url","position":0,"group_id":1,"action_code":301,"action_data":{"url":"https://azurdancestudio.ro"}}
Code: 404 
Raw: <html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>

**Comment by johngodley** [comment]
Can you go the Support page in Redirection and press the 'magic fixer' button?

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
